src/components/shared/layout/Filterbar.vue

```diff
@@ -1,12 +1,14 @@
 <template>
     <div class="ui secondary basic segment">
         <div class="ui text menu">
-            <div class="item">
+            <slot name="left-item">
```

Comment:
Were these changes requested? When we built this, it was to promote consistency with how a filterbar would appear all over CC, just the search bar on the left, all other options on the right and I think the idea was that it should always have a search bar even if it is a frontend filter?

Reply:
I made the changes as initially i planned to use it on the recurring tab of journals which doesn't require searches however i am not using it now, Product is open with this one, and is happy to go with your preference. i can roll back if you prefer, let me know. Thanks

Reply:
I think we should roll this back if there isn't a use case for it. 

When building things into the style guide, we aren't just adding components to a repo for reusability, we are setting a precedence and setting out the parameters on how a component can be used.

Until Product ask for a filter bar without a search, I believe we are setting a dangerous precedent by effectively saying this component can be used in such a way.